assets/queries/terraform/aws/s3_bucket_policy_accepts_http_requests/test/negative5.tf

```
@@ -0,0 +1,30 @@
+resource "aws_s3_bucket" "b" {
+  bucket = "my-tf-test-bucket"
+}
+
+resource "aws_s3_bucket_policy" "b" {
+  bucket = aws_s3_bucket.b.id
+  policy = data.aws_iam_policy_document.b.json
+}
+
+data "aws_iam_policy_document" "b" {
+  statement {
+    sid = "ForceSSLOnlyAccess"
+    effect = "Deny"
+    actions = ["s3:*"]
+    resource = [
+      aws_s3_bucket.b.arn
+    ]
+    
+    principals {
+      type = "*"
+      identifiers = ["*"]
+    }
+    
+    contidion {
```

Hi, @ezemedina!

Thank you so much for contributing to KICS!

I think there is a typo:
```suggestion
    condition {
```

PR #5415 improved this query recently. Can you merge this branch with the master and check if the samples cover your case, please? Thank you so much in advance 😊

---

Hello again, @ezemedina! Can you give us feedback, please?